cardano-cli/src/Cardano/CLI/EraBased/Run/Transaction.hs

```diff
@@ -998,8 +997,8 @@ runTransactionSubmitCmd
       , txFile
       } = do
   txFileOrPipe <- liftIO $ fileOrPipe txFile
-  InAnyCardanoEra era tx <- lift (readFileTx txFileOrPipe) & onLeft (left . TxCmdCddlError)
-  let txInMode = TxInMode era tx
+  InAnyShelleyBasedEra era tx <- lift (readFileTx txFileOrPipe) & onLeft (left . TxCmdCddlError)
```

Comment:
QA confirmed they use the "old" `cardano-cli byron` commands to submit txs. Unlikely anybody else is using this command to submit byron era txs.